src/App/index.js

```diff
@@ -34,9 +36,11 @@ export class App extends Component {
         <Router>
           <AppTemp id="homepage">
             <Switch>
+              <Route exact path="/" component={HomePage} />
               <Route exact path="/music" component={DefaultMusic} />
               <Route path="/music/buymusic" component={BuyMusic} />
               <Route path="/music/originals" component={DefaultOriginals} />
+              <Route path="/shop" component={ShopMain} />
```

Comment:
@Unesdala this route is not working for me, maybe you need to add the `exact` prop?

Reply:
@Unesdala nevermind, it works